app/api/models/user.py

```diff
@@ -47,7 +47,9 @@ def add_models_to_namespace(api_namespace):
         "bio": fields.String(required=True, description="User bio"),
         "location": fields.String(required=True, description="User location"),
         "occupation": fields.String(required=True, description="User occupation"),
-        "organization": fields.String(required=True, description="User organization"),
+        "current_organization": fields.String(
+            required=True, description="User current_organization"
```

Comment:
> It would be better if same convention is followed in `models/admin.py` and `models/user.py`.

models/admin, @devkapilbansal ? I can't see that file exist in MS backend base code?

> Same for other instances as well

I'm not sure I'm following, can you please elaborate? Which convention you are talking here?

Reply:
done